lib/driver.js

```diff
@@ -11,6 +11,7 @@ import { fs, tempDir } from 'appium-support';
 
 const APP_EXTENSION = '.apk';
 const DEVICE_PORT = 4724;
+const DEFAULT_ADB_PORT = 5037;
```

Comment:
Should this be exposed from `appium-adb` and just used here? Seems like info only it should know.